src/main/java/com/google/flourbot/entity/EntityModuleImplementation.java

```diff
@@ -78,8 +78,8 @@ public EntityModuleImplementation(DataStorage dataStorage) {
 
```

Comment:
L16: no * imports
L38: nit: there is no need to use the data structure name as suffix for the variable, although this is a common pattern. It is better to call it something like macroData, this way when the object type changes you don't have a problem. This applies to multiple variables in this file and in the project in general, but no need to fix all of them at this point.
L81: nit: columnValues please try to avoid abbreviations.

Reply:
Resolved all.